srcpkgs/libgaminggear/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'libgaminggear'
+pkgname=libgaminggear
+version=0.15.1
+revision=1
+build_style=cmake
+configure_args="-DINSTALL_CMAKE_MODULESDIR=/usr/share/cmake
+	-DINSTALL_PKGCONFIGDIR=/usr/share/pkgconfig"
+hostmakedepends="pkg-config glib-devel"
+makedepends="glib-devel gtk+-devel libnotify-devel sqlite-devel
+	libcanberra-devel"
+depends="libcanberra-devel libnotify-devel sqlite-devel"
+short_desc="Provides functionality for gaming input devices"
+maintainer="Alexander Gehrke <void@qwertyuiop.de>"
+license="GPL-2.0-only"
+homepage="https://sourceforge.net/projects/libgaminggear/"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.bz2"
+checksum=ffbd59c62e3107b09ec99f1e5147529c61931987abc7f86a140449b46388c549
+
+libgaminggear-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_${revision}"
```

Comment:
missing dependencies:

gtk+-devel libglib-devel libcanberra-devel libnotify-devel sqlite-devel

use `xpcdeps` from `xtools' to find dependencies